### PR TITLE
MAT-3572 metadata copyright

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-3572](https://jira.cms.gov/browse/MAT-3572)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  };
+
+  return (
+    <Form onSubmit={onFormSubmit} data-testid="measureCopyright">
+      <FormContent>
+        <div>
+          <FormField>
+            <FormFieldInner>
+              <FieldLabel htmlFor="measure-copyright">
+                Measure Copyright
```
Comment (adongare) [inline src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx:63]: I think label is just `Copyright`

Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx]: @adongare Thanks so much Ashok! I made the change.

**Review by adongare — APPROVED** [review]

File: src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx]: @adongare Hi Ashok! Is it ok if I resolve the conversation and merge the pull request? Thanks!

Comment (adongare) [reply src/components/editMeasure/measureDetails/measureCopyright/MeasureCopyright.tsx]: Hi Cecilia, please go ahead. Thanks!

**Pull request merged**